drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -570,7 +433,19 @@ static int parse_cfg(struct gps_config *cfg_src,
 	}
 
 	if (cfg_src->delete_agps_data) {
```

Comment:
I'm not willing to change it yet because the struct is defined in `nrf/include/drivers/gps.h`. I or someone else should make a PR which converts everything from GPS to GNSS and the forementioned variable name could be changed at the same time.

Reply:
I have understood that the intention was to deprecate and remove the GPS driver in favor of the new GNSS function based API. So probably there's no point in doing any unnecessary changes. Should the driver actually be marked as deprecated since NCS 1.6.0? There are several samples still using it, but those can be updated in NCS 1.7.0 to directly use the new GNSS API.

Reply:
Once there is no need to support the older modem firmware versions - pre-1.3.0 - that can be done.